src/lumigo_tracer/lumigo_utils.py

```diff
@@ -747,6 +747,22 @@ def lumigo_dumps(
     )
 
 
+def concat_old_body_to_new(old_body: Optional[str], new_body: bytes) -> str:
+    """
+    We have only a dumped body from the previous request,
+    so to concatenate the new body we should undo the lumigo_dumps.
+    Note that the old body is dumped bytes
```

Comment:
But the type is `old_body: Optional[str]` and not `old_body: Optional[bytes]`

Reply:
you right, dumping changes the data to str.
But dumping could fail, so this is Optional...